app/state_machines/flood_risk_engine/step_machine.rb

```diff
@@ -21,15 +21,17 @@ def current_step
       state.to_s
     end
 
+    # TODO: rename this method to something better. Style guide recommends
+    # not using set_ at start of method
+    # rubocop:disable Style/AccessorMethodName
     def set_step_as(step)
       restore!(step.to_sym)
     end
+    # rubocop:enable Style/AccessorMethodName
```

Comment:
I admit `go_to` was the first thing that sprang to my mind, but felt too "basic". I think `jump_to` will do. However, there is a little work to do to ensure the change has no effect, so I'll carry on with the merge and return to this later.